cmd/tracee-ebpf/internal/flags/flags-containers.go

```diff
@@ -0,0 +1,116 @@
+package flags
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/aquasecurity/tracee/pkg/containers/runtime"
+)
+
+func ContainersHelp() string {
+	return `Select which container runtimes to connect to for container events enrichment.
+By default, if no flag is passed, tracee will automatically detect installed runtimes by going through known runtime socket paths.
+
+Tracee will look for the following paths:
+1. Docker:     /var/run/docker.sock
+2. Containerd: /var/run/containerd/containerd.sock
+3. CRI-O:      /var/run/crio/crio.sock
+
+If runtimes are specified, only the ones passed through flags will be connected to through the provided socket file path.
+Supported runtimes are:
+
+1. CRI-O      (crio, cri-o)
+2. Containerd (containerd)
+3. Docker     (docker)
+
+Example:
+  --containers crio:/var/run/crio/crio.sock
+`
+}
+
+func contains(s []string, val string) bool {
+	for _, str := range s {
+		if str == val {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
I've seem this function repeated in previous PRs (although I don't remember where, and it might had another name), maybe we should put it somewhere under "utils" to avoid code duplication?

Reply:
We should add a generic version once we move to 1.18 probably.